lib/scss/components/event-filter-menu/stanford_events.event-filter-menu.scss

```diff
@@ -0,0 +1,162 @@
+@charset "UTF-8";
+@import '../../stanford_events.config';
+
+//
+// Filter menu
+//
+
+// Changing the layout for the filtered pages.
+@include grid-media-max('lg') {
+  .left-region {
+    &.flex-lg-3-of-12 {
+      flex: auto;
+      max-width: 100%;
+    }
+  }
+}
+
+.stanford-event-types {
+
+  // Filter open/close for xs-lg
+  @include grid-media-max('lg') {
+    @include modular-spacing('margin', null null 4 null);
+
+    & > .menu {
+      display: none;
+    }
+
+    .event-types__collapsable-menu {
+      &.show {
+        & + .menu {
+          display: block;
+        }
+      }
+    }
+  }
+
+
+  // Filter open/close for xl
+  @include grid-media('xl') {
+    & > .menu {
+      display: block;
+    }
+
+    .event-types__collapsable-menu {
+      &.show {
+        + .menu {
+          display: none;
+        }
+      }
+    }
+  }
+
+
+
```

Comment:
Extra space should be removed. Where you ad CodeClimate?

Reply:
all done